docs/data_types/raster.rst

```diff
@@ -103,10 +103,10 @@ Dimensions
 .. code-block:: python
 
     >>> my_rasters.raster_array_axis_physical_types
-    [('meta.obs.sequence',), ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon', 'time'), ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'), ('em.wl',)]
+    [('meta.obs.sequence',), (np.str_('custom:pos.helioprojective.lat'), np.str_('custom:pos.helioprojective.lon'), np.str_('time')), (np.str_('custom:pos.helioprojective.lat'), np.str_('custom:pos.helioprojective.lon')), (np.str_('em.wl'),)]
```

Comment:
booo

Reply:
I will wait for the ndcube PR to be merged so I can get a release out and then I can undo this.